build.gradle.kts

```diff
@@ -78,7 +83,7 @@ configurations {
   implementation.configure {
     exclude(module = "spring-boot-starter-web")
     exclude("org.apache.tomcat")
-    exclude("ch.qos.logback")
+    exclude(group = "org.slf4j", module = "slf4j-simple")
```

Comment:
Is it ok replacing Logback with SL4J-simple?

Reply:
Yes, here we are excluding SL4J-simple (transient dependency of openapi generator) and using only logback as the only log provider